templates/Layout/ProductHolder.ss

```diff
@@ -28,6 +28,7 @@
                     <div class="unit size3of4 productSummaryText">
                         <h3><a href="{$Link}" title="<% if $ReceiptTitle %>{$ReceiptTitle}<% else %>{$Title}<% end_if %>"><% if $ReceiptTitle %>{$ReceiptTitle.LimitCharacters(48)}<% else %>{$Title.LimitCharacters(48)}<% end_if %></a></h3>
                         <b>$Price.Nice</b>
+                        <br>$ProductRating
```

Comment:
The only thought here would be wrapping this in an `<% if $ProductRating %>` so we don't have the `<br>` in there if there are no product ratings. Similarly the items in ProductPage.ss